### DISCUSSION TITLE
Settings: Add a shortcut icon for defining hotkeys

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#209] [Ideas - done & released]
See these ![kuva](https://user-images.githubusercontent.com/13002142/165547712-9e8cacaf-8e9b-47fb-a8e4-9e9234991e76.png) icons:

![kuva](https://user-images.githubusercontent.com/13002142/165547553-172c71c3-f6fa-4156-8ed8-a71d1fd58207.png)

Click on an icon, and the hotkey settings will be opened, showing only that particular command:

![kuva](https://user-images.githubusercontent.com/13002142/165548094-6f8ccfa2-12be-47dd-8072-5032ca6dd436.png)

I'm using a few lines of @pjeby's code from here: https://github.com/pjeby/hotkey-helper/blob/master/src/plugin.js

**Comment by Taitava** [comment]
@pjeby is it ok that I'm borrowing your code? 🙂 The borrowed code can be seen in the following commit: https://github.com/Taitava/obsidian-shellcommands/commit/6e31d42431d4209d0672aeecc551d11b5c0af95e (file `CreateShellCommandField.ts`).

**Reply by pjeby** [reply]
No problem at all.  Go for it!

**Comment by Taitava** [comment]
# Released
